src/store/modules/ui.store.js

```diff
@@ -412,13 +418,22 @@ export default {
                 commit('removeError', { error, dispatcher })
             }
         },
-        addWarnings({ commit }, { warnings, dispatcher }) {
+        addWarnings({ commit, state }, { warnings, dispatcher }) {
             if (
                 warnings instanceof Array &&
-                warnings.filter((warning) => warning instanceof WarningMessage).length ===
-                    warnings.length
+                warnings.every((warning) => warning instanceof WarningMessage)
             ) {
-                commit('addWarnings', { warnings, dispatcher })
+                warnings = warnings.filter(
+                    (warning) =>
+                        // we only add the warnings that are not existing within the store
+                        ![...state.warnings].some((otherWarning) => {
+                            warning.isEquals(otherWarning)
+                        })
+                )
+                log.error(warnings)
```

Comment:
perhaps `log.warn`?

Reply:
this looks like a forgotten log while developing. I'll remove it entirely. Thanks :)